python/huggingfaceserver/huggingfaceserver/vllm/utils.py

```diff
@@ -54,6 +54,41 @@ def infer_vllm_supported_from_model_architecture(
 def maybe_add_vllm_cli_parser(parser: ArgumentParser) -> ArgumentParser:
     if not _vllm:
         return parser
+
+    parser.add_argument(
+        "--enable-auto-tool-choice",
```

Comment:
vLLM should have already defined these arguments?

Reply:
Apparently not in the engine args: https://github.com/vllm-project/vllm/blob/main/vllm/engine/arg_utils.py#L209